home.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CineOnline</title>
+    <link href="css/estilos.css" rel="stylesheet"/>
+</head>
+<body>
+
+<!--Cabecero de la página principal "HOME"-->
+<header>
+    <nav class="nav-fixed">
+        <img id="logo" src="imagenes/cine_adobespark (1).png" alt="Logo Cinemath"/>
+        <h3 id="id">CineOnline</h3>
+        <a href="home.html">Home</a>

```

Comment:
Mantener la consistencia en el lenguaje, si vamos a utilizar Español que todos los textos sean en Español.
```suggestion
        <a href="home.html">Inicio</a>

```